### PR TITLE
chore: check flake inputs as a non-mandatory CI step

### PR DESCRIPTION
**Pull request by dpc**
Just to remaind us to update nixpkgs from time to time?

### CODE REVIEW

**Comment by douglaz** [comment]
Update nixpkgs?

**Comment by dpc** [comment]
> Update nixpkgs?

I needed to admire how it fails.

File: .github/workflows/ci-nix.yml [hunk 1]
```diff
@@ -19,6 +19,16 @@ on:
...
 jobs:
+  self-care:
+    name: Flake self-check
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Check Nix flake inputs
+        uses: DeterminateSystems/flake-checker-action@v5
+        with:
+          fail-mode: true
```
Comment (Maan2003) [inline .github/workflows/ci-nix.yml:30]: Will this block merging?

Comment (dpc) [reply .github/workflows/ci-nix.yml:30]: Nope

Comment (dpc) [reply .github/workflows/ci-nix.yml:30]: Only the workflows explicitly listed in github settings for the repo are required.

**Review by Maan2003 — APPROVED** [review]

**Pull request merged**